homeassistant/components/knx/schema.py

```diff
@@ -24,6 +25,32 @@
     ColorTempModes,
 )
 
+##################
+# KNX VALIDATORS
+##################
+
+ga_validator = vol.Any(
+    cv.matches_regex(GroupAddress.ADDRESS_RE),
+    vol.All(int, vol.Range(min=1, max=65535)),
+    msg="value does not match pattern for KNX group address '<main>/<middle>/<sub>', '<main>/<sub>' or '<free>' (eg.'1/2/3', '9/234', '123')",
+)
+
+ia_validator = vol.Any(
+    cv.matches_regex(IndividualAddress.ADDRESS_RE),
+    vol.All(int, vol.Range(min=1, max=65535)),
+    msg="value does not match pattern for KNX individual address '<area>.<line>.<device>' (eg.'1.1.100')",
+)
+sync_state_validator = vol.Any(
```

Comment:
```suggestion

sync_state_validator = vol.Any(
```

Add an empty line as above